gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -170,18 +170,17 @@ private ChannelAndRequestCounter buildManagedChannel() {
     return new ChannelAndRequestCounter(channel, counter);
   }
 
+  public boolean isStubBroken(Status.Code statusCode) {
```

Comment:
should we make the method static ? thoughts?

Reply:
It depends on non-static field.

Reply:
nvm, done.